src/sinol_make/__init__.py

```diff
@@ -1,12 +1,12 @@
-import argparse
+import argparse, pkg_resources
 from sinol_make import util
 
 def configure_parsers():
 	parser = argparse.ArgumentParser(
 		prog='sinol-make',
 		description='Tool for creating and testing sio2 tasks',
 	)
-	parser.add_argument("-v", "--version", action="version", version="%(prog)s 0.0.1")
+	parser.add_argument("-v", "--version", action="version", version="%(prog)s " + pkg_resources.get_distribution('sinol_make').version)
```

Comment:
Is the `s` in `%(prog)s` on purpose or is it a typo?

Reply:
No, it's there to specify that the `prog` should be converted to string

Reply:
Ok, I see. So the output will be `sinol-make: 0.0.1`?